.github/workflows/build-and-test.yaml

```diff
@@ -178,7 +207,36 @@ jobs:
         Import-Module "$env:ChocolateyInstall\helpers\chocolateyProfile.psm1"
         refreshenv
 
-        choco install make && make build-docker-images-windows
+        choco install make && make build-docker-images-windows-2019
+
+  buildWindows2022Docker:
```

Comment:
same here as well, because i see that only change is runs-on which machine and the make target, try to verify if you can make this work on single job.